content/blog/findings-when-coverting-tests-to-pester5.md

```diff
@@ -0,0 +1,303 @@
+---
+title: "Findings when converting tests to Pester 5"
+date: 2020-05-17T05:00:00+01:00
+type: "post"
+draft: false
+author: johlju
+---
+
+If you have any questions or suggestions around this blog post then please
+reach out to `@johlju` in the Virtual PowerShell User Group [#DSC channel](https://dsccommunity.org/community/contact/).
+
+## Pester
+
+Pester is used throughout the DSC modules and their common modules to ensure
+that we maintain quality for all the contributions by both testing the new
+contribution (the tests added or changed in the contribution) and the
+new contribution together with the existing code.
+
+Pester is about to be released in a new version 5.0.0 with breaking changes
```

Comment:
"A new major version of Pester is about to be released, bringing improvements in some areas along with breaking changes compared to the previous v4 versions."
"In this post I will explain what may be needed to convert your existing tests from v4 to v5."